src/main/resources/xsd/actionInstruction.xsd

```diff
@@ -159,6 +159,7 @@
                     <xs:element name="reason" type="xs:string"/>
                     <xs:element name="caseId" type="xs:string" minOccurs="0" maxOccurs="1"/>
                     <xs:element name="caseRef" type="xs:string" minOccurs="0" maxOccurs="1"/>
+                    <xs:element name="addressType" type="xs:string" minOccurs="0" maxOccurs="1"/>
```

Comment:
has this change been accepted by Dave Richards etc?

Reply:
Checked with Olu and it is expected to be here.

Reply:
okay 🌮